pages/offres/partage-suggestion.tsx

```diff
@@ -0,0 +1,176 @@
+import { Etape } from 'components/ui/Form/Etape'
+import IconComponent, { IconName } from 'components/ui/IconComponent'
+import { BaseJeune } from 'interfaces/jeune'
+import { TypeOffre } from 'interfaces/offre'
+import { PageProps } from 'interfaces/pageProps'
+import { LocaliteType } from 'interfaces/referentiel'
+import { QueryParam, QueryValue } from 'referentiel/queryParam'
+import { JeunesService } from 'services/jeunes.service'
+import { SuggestionsService } from 'services/suggestions.service'
+import { withMandatorySessionOrRedirect } from 'utils/auth/withMandatorySessionOrRedirect'
+import { useDependance } from 'utils/injectionDependances'
+import withDependance from 'utils/injectionDependances/withDependance'
+
+type PartageSuggestionProps = PageProps & {
+  jeunes: BaseJeune[]
+  type: TypeOffre
+  titre: string
+  motsCles: string
+  typeLocalite: LocaliteType
+  labelLocalite: string
+  codeLocalite: string
+  withoutChat: true
+  returnTo: string
+}
+
+function PartageSuggestion({
+  jeunes,
+  type,
+  titre,
+  motsCles,
+  typeLocalite,
+  labelLocalite,
+  codeLocalite,
+  returnTo,
+}: PartageSuggestionProps) {
+  const suggestionsService =
+    useDependance<SuggestionsService>('suggestionsService')
+  const router = useRouter()
+  const [idsDestinataires, setIdsDestinataires] = useState<
+    RequiredValue<string[]>
+  >({ value: [] })
+  const [isPartageEnCours, setIsPartageEnCours] = useState<boolean>(false)
+
+  const formIsValid = useMemo(
+    () => idsDestinataires.value.length > 0,
+    [idsDestinataires]
+  )
+
+  function updateIdsDestinataires(selectedIds: string[]) {
+    setIdsDestinataires({
+      value: selectedIds,
+      error: !selectedIds.length
+        ? "Aucun bénéficiaire n'est renseigné. Veuillez sélectionner au moins un bénéficiaire."
+        : undefined,
+    })
+  }
+
+  async function partagerCriteresRecherche(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault()
+    if (!formIsValid) return
+
+    setIsPartageEnCours(true)
+
+    try {
+      await suggestionsService.postSuggestionOffreEmploi(
+        idsDestinataires.value,
+        titre,
+        motsCles,
+        labelLocalite,
+        typeLocalite === 'DEPARTEMENT' ? codeLocalite : undefined,
+        typeLocalite === 'COMMUNE' ? codeLocalite : undefined
+      )
+      await router.push({
+        pathname: '/recherche-offres',
+        query: { [QueryParam.partageCriteres]: QueryValue.succes },
+      })
+    } finally {
+      setIsPartageEnCours(false)
+    }
+  }
+
+  return (
+    <>
+      <SuggestionCard
+        type={type}
+        titre={titre}
+        motsCles={motsCles}
+        labelLocalite={labelLocalite}
+      />
+      <p className='mt-8'>
+        Ces critères apparaitrons dans la section favoris, catégorie recherches
+        sauvegardées,du bénéficiaire de l’application CEJ.
+      </p>
+      <form onSubmit={partagerCriteresRecherche} className='mt-8'>
+        <Etape numero={1} titre='Bénéficiaires'>
```

Comment:
on reste sur "Destinataires" a priori? cf. notion et maquette
```suggestion
        <Etape numero={1} titre='Destinataires'>
```